packages/react-table/src/demos/Table.md

```diff
@@ -2397,52 +2394,77 @@ ComposableTableSortable = () => {
             </Text>
           </TextContent>
         </PageSection>
-        <PageSection isFilled>
-          {tableToolbar}
-          <TableComposable aria-label="Sortable Table">
-            <Thead>
-              <Tr>
-                {columns.map((column, columnIndex) => {
-                  const sortParams = {
-                    sort: {
-                      sortBy: {
-                        index: activeSortIndex,
-                        direction: activeSortDirection
-                      },
-                      onSort,
-                      columnIndex
-                    }
-                  };
-                  const infoParams =
-                    columnIndex === 2
-                      ? {
-                          info: {
-                            tooltip: 'More information'
-                          }
-                        }
-                      : {};
-                  return (
-                    <Th key={columnIndex} {...sortParams} {...infoParams}>
-                      {column}
-                    </Th>
-                  );
-                })}
-              </Tr>
-            </Thead>
-            <Tbody>
-              {rows.map((row, rowIndex) => (
-                <Tr key={rowIndex}>
-                  {row.map((cell, cellIndex) => (
-                    <Td key={`${rowIndex}_${cellIndex}`} dataLabel={columns[cellIndex]}>
-                      {cell}
-                    </Td>
-                  ))}
+        <PageSection
+          padding={{
+            default: 'noPadding',
+            xl: 'padding'
+          }}
+        >
+          <Card component="div">
+            {tableToolbar}
+            <TableComposable aria-label="Sortable Table">
+              <Thead>
+                <Tr>
+                  <Th />
+                  {columns.map((column, columnIndex) => {
+                    const sortParams = {
+                      sort: {
+                        sortBy: {
+                          index: activeSortIndex,
+                          direction: activeSortDirection
+                        },
+                        onSort,
+                        columnIndex
+                      }
+                    };
+                    return (
+                      <Th key={columnIndex} {...sortParams}>
```

Comment:
The core demo starts off with the first column header `<th>` marked as sorted. The react demo content is also visually sorted by the first column, but the column header isn't marked as sorted.

Reply:
Nice catch - updated!